.github/ISSUE_TEMPLATE/tutorial_request.yaml

```diff
@@ -0,0 +1,37 @@
+name: GoaT Tutorial Request
+description: Unsure about a query or feature? Request a tutorial for GoaT Wiki or Youtube Channel.
+title: New Tutorial [TITLE]
+labels: "project:goat :goat:, process:documentation, stack:ui, stack:data"
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Use this form to request a tutorial on GoaT. Thanks for taking the time to contribute. 
+  - type: checkboxes
+    id: confirmation
+    attributes:
+      label: tutorial type
+      description: Use this field to inform if you would like to request a new tutorial  or refine an existing tutorial on [GoaT-Wiki](https://github.com/genomehubs/goat-data/wiki) or [YouTube channel](https://www.youtube.com/@genomesonatree)
```

Comment:
delete "to inform"
tutorial or refine an existing tutorial on -> tutorial, or an update to an existing tutorial, on the